TicTacToeAppTests/ViewControllerSpec.swift

```diff
@@ -0,0 +1,101 @@
+import Quick
+import Nimble
+import TicTacToeApp
+
+class ViewControllerSpec: QuickSpec {
+  override func spec() {
+
+    describe("ViewController") {
+
+      it("disables button after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.enabled).to(beFalse())
+      }
+
+      it("changes button title after making a move") {
+        let controller = ViewController()
+        let button = UIButton()
+
+        controller.makeMove(button)
+
+        expect(button.currentTitle).notTo(beNil())
+      }
+
+      it("changes button title to alternating player marks after each move") {
+        let controller = ViewController()
+        let firstMoveButton = UIButton()
+        let secondMoveButton = UIButton()
+        let thirdMoveButton = UIButton()
+
+        controller.makeMove(firstMoveButton)
+        controller.makeMove(secondMoveButton)
+        controller.makeMove(thirdMoveButton)
+
+        expect(firstMoveButton.currentTitle).to(equal("X"))
+        expect(secondMoveButton.currentTitle).to(equal("O"))
+        expect(thirdMoveButton.currentTitle).to(equal("X"))
+      }
+
+      it("removes board button titles when the game is reset") {
+        let controller = ViewController()
+        controller.boardButtons = [
```

Comment:
Not for all tests, no. Do you think it's an unreasonable amount of overhead? I thought it would simplify things to be able to assume all the buttons are initialized for each test context.